lib/src/base/paged_layout_builder.dart

```diff
@@ -113,7 +113,12 @@ class _PagedLayoutBuilderState<PageKeyType, ItemType>
       // We make sure to only schedule the fetch after the current build is done.
       // This is important to prevent recursive builds.
       () => WidgetsBinding.instance
-          .addPostFrameCallback((_) => widget.fetchNextPage());
+          .addPostFrameCallback((_) {
+            if(!mounted) {
+              return;
+            }
```

Comment:
Please use the short form, like so:
```suggestion
            if(!mounted) return;
```